relay-event-normalization/src/normalize/span/ai.rs

```diff
@@ -207,3 +259,104 @@ fn is_ai_span(span: &Span) -> bool {
         .value()
         .is_some_and(|op| op.starts_with("ai.") || op.starts_with("gen_ai."))
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_calculate_cost_no_tokens() {
+        let cost = calculate_costs(
+            &ModelCostV2 {
+                input_per_token: 1.0,
+                output_per_token: 1.0,
+                output_reasoning_per_token: 1.0,
+                input_cached_per_token: 1.0,
+            },
+            UsedTokens::from(&SpanData::default()),
```

Comment:
If you derive `Default` for `UsedTokens`:
```suggestion
            UsedTokens::default(),
```

Reply:
I don't think `UsedTokens` should semantically have a `Default`, also this was a bit on purpose to make sure the conversion works properly (since there are unwrap_or's in there).